src/main/resources/i18n/Validation.properties

```diff
@@ -98,3 +100,6 @@ ArgumentValidationUtil.handleNotObjectError=Validation error ({0}) : argument ''
 ArgumentValidationUtil.handleMissingFieldsError=Validation error ({0}) : argument ''{1}'' with value ''{2}'' is missing required fields ''{3}''
 # suppress inspection "UnusedProperty"
 ArgumentValidationUtil.handleExtraFieldError=Validation error ({0}) : argument ''{1}'' with value ''{2}'' contains a field not in ''{3}'': ''{4}''
+
+DirectiveMisplaced.notAllowedOperationRootLevel=Validation error ({0}) : Directive ''{1}'' is not allowed on root of operation ''{2}''
+DirectiveMisplaced.notAllowedOperation=Validation error ({0}) : Directive ''{1}'' is not allowed on operation ''{2}''
```

Comment:
I think this should be much more specific (eg to the rule)

`The ''{{1}}` directive is not allowed to be used on a subscription operation`